### PR TITLE
all integration-tests run (with the exception of test_cisc)

### PR DESCRIPTION
**Pull request by ineiti**
Implements #537 - depends on https://github.com/dedis/cosi/pull/54

### CODE REVIEW

**Comment by Liamsi** [comment]
General remark: What I'm missing in this PR (or in the related issue #537) is a short description what wasn't working (and what is now; after this PR and https://github.com/dedis/cosi/pull/54). One can only guess from skimming through the changes.

File: app/lib/server/server.go [hunk 1]
```diff
@@ -228,6 +228,7 @@ func CheckServers(g *config.Group) error {
 	}
 
 	if !success {
+		log.Error("at least one failed")
```
Comment (nikkolasg) [inline app/lib/server/server.go]: Why logging ? If it's already printed at the screen for each failing line + a returning error at the end of this function...

**Comment by ineiti** [comment]
@Liamsi - better description now?

**Comment by nikkolasg** [comment]
OK to merge.

**Comment by Liamsi** [comment]
> @Liamsi - better description now?

Yep, thanks!

**Pull request merged**